drupal/sites/all/libraries/ckeditor/README.md

```diff
@@ -1,7 +1,7 @@
 CKEditor 4
 ==========
 
-Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+Copyright (c) 2003-2023, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
This is a 3rd-party file and must not be changed

Reply:
reverted back